.github/workflows/dev-ci-cd.yml

```
@@ -0,0 +1,96 @@
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
+
+  deploy-dev:
+    needs: build-and-push
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    environment: development
+    if: github.ref == 'refs/heads/develop'
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)